### PR TITLE
MDEXP-305 Fix generated mrc files and directories are not being deleted

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
## Purpose
https://issues.folio.org/browse/MDEXP-305

## Approach
While investigating the issue, I saw that it is unreproducible on test/scratch envs. Investigated logs from bugfest-goldenroad, no logs about cleanup process - so the cleanup job is not running at all, or stuck. 
Decided to change mechanism of the cleanup process to provide a new endpoint, and use okapi timer to retrieve this API.
Also, it will give us the opportunity to manually start the cleanup process.
Updated rancher configs to helm configs with version: 0.1.30

### CODE REVIEW

**Comment by andrii-novytskyi** [comment]
@folio-org/dataexportbackend The rest assured tests are present in PR, but for some reason sonar thinking that new method in controller not covered, does anybody knows what is the reason?

File: src/main/java/org/folio/rest/impl/ModTenantAPI.java [hunk 1]
```diff
@@ -35,28 +29,13 @@ public ModTenantAPI() { //NOSONAR
   public void postTenant(TenantAttributes entity, Map<String, String> headers, Handler<AsyncResult<Response>> handlers, Context context) {
     super.postTenant(entity, headers, asyncResult -> {
       if (asyncResult.failed()) {
+        LOGGER.error("Post tenant is failed, cause: " + asyncResult.cause().getMessage());
```
Comment (KVupp) [inline src/main/java/org/folio/rest/impl/ModTenantAPI.java:32]:
```suggestion
        LOGGER.error("Post tenant failed, cause: " + asyncResult.cause().getMessage());
```

File: src/main/java/org/folio/rest/impl/ModTenantAPI.java [hunk 2]
```diff
@@ -35,28 +29,13 @@ public ModTenantAPI() { //NOSONAR
...
         handlers.handle(asyncResult);
       } else {
-        initStorageCleanupService(headers, context);
+        LOGGER.info("Post tenant is complete successfully");
```
Comment (KVupp) [inline src/main/java/org/folio/rest/impl/ModTenantAPI.java:35]:
```suggestion
        LOGGER.info("Post tenant is completed successfully");
```

File: src/main/java/org/folio/service/file/cleanup/StorageCleanupServiceImpl.java [hunk 3]
```diff
@@ -44,6 +46,7 @@ private Date getFileDefinitionExpirationDate() {
   }
 
   private Future<CompositeFuture> deleteExpiredFilesAndRelatedFileDefinitions(List<FileDefinition> fileDefinitions, String tenantId) {
+    LOGGER.info(format("Start process of removing files and file definitions, number of file definitions to clean up: %s", fileDefinitions.size()));
```
Comment (KVupp) [inline src/main/java/org/folio/service/file/cleanup/StorageCleanupServiceImpl.java:49]: Ideally we should be using the parameterized logging offered by log4j : https://logging.apache.org/log4j/2.x/performance.html#asyncLoggingWithParams

Comment (andrii-novytskyi) [reply src/main/java/org/folio/service/file/cleanup/StorageCleanupServiceImpl.java]: @KVupp thanks, done

**Review by KVupp — APPROVED** [review]

**Review by Igor-Gorchakov — APPROVED** [review]

**Pull request merged**